postgrest.cabal

```diff
@@ -29,17 +29,17 @@ executable postgrest
     -rtsopts
     "-with-rtsopts=-N -I2"
   default-language:    Haskell2010
-  build-depends:       aeson (>= 0.8 && < 0.10) || (>= 0.11 && < 0.12)
+  build-depends:       aeson (>= 0.11 && < 0.12) || (>= 1.0 && < 1.1)
                      , auto-update
                      , base >= 4.8 && < 6
                      , bytestring
-                     , bytestring-tree-builder == 0.2.7
+                     , bytestring-tree-builder == 0.2.7.1
                      , case-insensitive
                      , cassava
                      , containers
                      , contravariant
                      , errors
-                     , hasql == 0.19.12
+                     , hasql == 0.19.15.2
```

Comment:
I think I made the direct match so that `packdeps` would warn me when hasql has been upgraded beyond the strict bounds. But I guess with your proposal it still would warn me and it would be a more meaningful version change. So good idea.